### PR TITLE
Adds Layout component and begins grid structure

### LINKED ISSUE #4: Layout/Page Shell

### PR DESCRIPTION
**Pull request by campbellkearns**
closes #4

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ module.exports = {
       'serif': [...defaultTheme.fontFamily.serif],
       'mono': [...defaultTheme.fontFamily.mono],
     },
-    extend: {},
+    extend: {
```
Comment (saihaj) [inline tailwind.config.js:14]: Do we need this?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+import Layout from '../components/layout'
 import styles from '../styles/Home.module.css'
 
-const Home = () => {
+const Home = (): JSX.Element => {
   return (
-    <div className={styles.container}>
+    <Layout>
       <Head>
         <title>ScribeDAO</title>
```
Comment (saihaj) [inline src/pages/index.tsx:9]: We can move `<title>` to Layout to where `ScribeDAO` is default and then if you pass a title prop to `<Layout>` it becomes `PROP_VALUE | ScribeDAO`

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import Head from 'next/head'
+import Layout from '../components/layout'
 import styles from '../styles/Home.module.css'
 
-const Home = () => {
+const Home = (): JSX.Element => {
```
Comment (saihaj) [inline src/pages/index.tsx:5]: Is TS not inferring return types?

**Review by saihaj — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (campbellkearns) [reply tailwind.config.js:14]: nope not yet, left behind after an experiment - thanks

File: src/pages/index.tsx (hunk 3 above)
Comment (campbellkearns) [reply src/pages/index.tsx:5]: Too explicit & not necessary, you're right. Removed

File: src/pages/index.tsx (hunk 2 above)
Comment (campbellkearns) [reply src/pages/index.tsx:9]: Sick, done. future state, i think a `siteMetadata` file could make sense for static things like the title and certain copy.

**Pull request merged**